velox/functions/sparksql/Bitwise.cpp

```diff
@@ -54,6 +56,16 @@ void registerBitwiseFunctions(const std::string& prefix) {
   registerBinaryIntegral<BitwiseOrFunction>({prefix + "bitwise_or"});
   registerBinaryIntegral<ShiftLeftFunction>({prefix + "shiftleft"});
   registerBinaryIntegral<ShiftRightFunction>({prefix + "shiftright"});
+
+ registerFunction<ShiftLeftFunction, int32_t, int32_t, int32_t>({prefix + "shiftleft"});
+ registerFunction<ShiftLeftFunction, int32_t, int32_t, int64_t>({prefix + "shiftleft"});
```

Comment:
It looks the last type can only be int32, for specifying bits num. So I think the below code is enough for shiftLeft.
```
registerFunction<ShiftLeftFunction, int32_t, int32_t, int32_t>({prefix + "shiftleft"});
registerFunction<ShiftLeftFunction, int64_t, int64_t, int32_t>({prefix + "shiftleft"});
```